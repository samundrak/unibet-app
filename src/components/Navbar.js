import React from 'react';

const Navbar = () => (
  <ul>
    <li>
      <a href="instructions/index.html">Test instructions</a>
    </li>
  </ul>
);
export default Navbar;
