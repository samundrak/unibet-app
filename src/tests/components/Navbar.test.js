import React from 'react';
import { render } from 'react-dom';
import Navbar from '../../components/Navbar';

describe('test Navbar', () => {
  it('should test snapshot', () => {
    const container = document.createElement('div');
    render(<Navbar />, container);
    expect(container.innerHTML).toMatchSnapshot();
  });
});
