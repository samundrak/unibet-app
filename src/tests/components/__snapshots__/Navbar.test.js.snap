// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Navbar should test snapshot 1`] = `"<ul><li><a href=\\"instructions/index.html\\">Test instructions</a></li></ul>"`;
